src/stores/cluster/index.js

```diff
@@ -304,6 +299,22 @@ export default class ClusterStore extends Store {
   registerStore = (name, store) => {
     this.store[name] = store;
   };
+
+  @action
+  updateEnv = async clusterIds => {
+    const clusterId = clusterIds[0];
+    const result = await this.request.patch('clusters/update_env', {
+      cluster_id: clusterId,
+      env: this.clusterEnv
+    });
+    if (get(result, 'cluster_id')) {
+      this.hideModal();
+      // for refresh env
+      await this.getStore('clusterDetail').fetch(clusterIds[0]);
+      await this.fetchJobs(clusterId);
+      this.success(ts('Update cluster environment successfully.'));
```

Comment:
remove `ts()`, just call `this.success(..)`

Reply:
ok